back/engines/commercial/insights/spec/models/insights/category_spec.rb

```diff
@@ -2,43 +2,23 @@
 
```

Comment:
Adding tests for the new `source` association and simplifying a lot of tests along the way.